test/esm/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@babel/test-esm",
```

Comment:
This package is not visible to lerna. It is handled by yarn so we can import `@babel/runtime` from the workspace.